SMLHelper/SMLHelper.csproj

```
@@ -72,12 +72,10 @@
       <Private>False</Private>
     </Reference>
     <Reference Include="Assembly-CSharp-firstpass_publicized">
-      <HintPath>$(Dependencies)\Assembly-CSharp-firstpass_publicized.dll</HintPath>
-      <Private>False</Private>
+      <HintPath>D:\SteamLibrary\steamapps\common\Subnautica\Subnautica_Data\Managed\publicized_assemblies\Assembly-CSharp-firstpass_publicized.dll</HintPath>
```

```suggestion
      <HintPath>$(Dependencies)\Assembly-CSharp-firstpass_publicized.dll</HintPath>
      <Private>False</Private>
```

You should move these DLLs into the dependencies folder of the solution, rather than hard-coding their path into the repository to a file location that may be different for every other contributor.